src/main/java/org/broadinstitute/hellbender/engine/ReferenceContext.java

```diff
@@ -352,7 +352,7 @@ private int trimToContigLength(final String contig, final int end){
 
         final SAMSequenceRecord sequence = dataSource.getSequenceDictionary().getSequence(contig);
         if ( sequence == null ) {
-            throw new UserException("Given reference file does not have data at the requested contig(" + contig + ")!");
+            throw new UserException.NoDataAtRequestedContig("Given reference file does not have data at the requested contig(" + contig + ")!");
```

Comment:
With the current implementation of `NoDataAtRequestedContig` you should only pass `contig` to the constructor. Otherwise the message will be weird.

Reply:
Yup.  Fixed!